gettingstarted/InstallingEquella.md

```diff
@@ -3,39 +3,46 @@
 # Installing openEQUELLA
 
 ## Environment setup
-Assuming you are building and installing openEQUELLA in the same environment, the following should be setup.  Details can be found in the [openEQUELLA Dev Setup](https://github.com/equella/Equella#download-required-software) guide.
+The following dependencies are required to successfully install openEQUELLA.
 
-- Java 8 (Oracle)
-- ImageMagick
-- Libav
-- Database
+- [Java 8 JDK](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) 
 
```

Comment:
Can you add the latest known working version of Java 8 with oE here? - Maybe just say the 'latest Java 8' JDK.

Reply:
Oracle is not mandatory i guess!! I have installed oE with openJdk 8 and it works fine.

Reply:
Yes, modern oEQ is happy with Open JDK 8.

Reply:
This is now being addressed by new page from #54 and hopefully with PRs for #66.